src/bmi_cfe.c

```diff
@@ -3325,11 +3326,12 @@ extern void mass_balance_check(cfe_state_struct* cfe_ptr){
     giuh_residual = cfe_ptr->vol_struct.vol_runoff - cfe_ptr->vol_struct.vol_out_surface - cfe_ptr->vol_struct.vol_end_surface -
                     cfe_ptr->vol_struct.vol_runon_infilt;
```

Comment:
Question: is vol_struct>vol_et_from_retention_depth included in cfe_ptr->vol_struct.vol_end_surfaced?   Secong question, should this mass balance equation on line 3325 be:  giuh_residual = cfe_ptr->vol_struct.vol_runoff - cfe_ptr->vol_struct.vol_out_surface - cfe_ptr->vol_struct.vol_end_surface - cfe_ptr->vol_struct.vol_runon_infilt - cfe_ptr->vol-struct.vol_et_from_retention_depth; ?